org.adempiere.ui.zk/WEB-INF/src/web/js/html2canvas/zk.wpd

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <package name="html2canvas" language="xul/html">
-	<script src="html2canvas.js" />
+	<script src="html2canvas.min.js" />
```

Comment:
This is not correct. With Zk naming convention, we have to rename html2canvas.js to html2canvas.src.js (for source) and html2canvas.min.js to html2canvas.js

The convention is fix and can't be changed.

Reply:
ah i see. at first, i wanna use html2canvas-pro but it make the build error. can we use html2canvas-pro? it's better to make different with old html2canvas library.

Reply:
of course, the naming is not important.